pom.xml

```diff
@@ -181,6 +176,21 @@
             <version>${commons-io.version}</version>
             <scope>test</scope>
         </dependency>
+
+        <dependency>
```

Comment:
I love testcontainers but what's the advantage of using them over hsqldb in our case?
Let's not add these dependencies in this PR since they are not used.

Reply:
This is a intermediate PR for starting #7. You may check for the example of a test. https://github.com/eshepelyuk/avro-schema-generator/blob/issue_7/src/test/java/com/at/avro/integration/PostgresqlTest.java

Basically i am going to use real db for testing postgres specific types mapping, instead of creating synthetic tests based on mocks.

Reply:
Got it. Since you already have the test let's keep the dependencies. Just didn't want to merge something that's not used.